### PR TITLE
docs: TUI install and connect to k8s installation

### LINKED ISSUE #5997: fix: docs: TUI install and connect to k8s installation -autocreated
This issue was created because no open issues were referenced in your PR (#5996).

### PR DESCRIPTION
**Pull request by misba7**
### Prepare

- [ ] Read [PR guidelines](https://github.com/JanssenProject/jans/blob/main/docs/CONTRIBUTING.md#prs)
- [ ] Read [license information](https://github.com/JanssenProject/jans/blob/main/LICENSE)

-------------------

#### Target issue
  https://github.com/JanssenProject/jans/issues/5995
  
  
  
closes #issue-number-here

#### Implementation Details
  

-------------------
### Test and Document the changes
- [ ] Static code analysis has been run locally and issues have been fixed
- [ ] Relevant unit and integration tests have been added/updated
- [ ] Relevant documentation has been updated if any (i.e. user guides, installation and configuration guides, technical design docs etc)

Closes #5997, #5995

### CODE REVIEW

**Comment by mo-auto** [comment]
Error: Hi @misba7, You did not reference an open issue in your [PR](5996). I attempted to create an [issue](#5997) for you.    
 Please update that [issues'](#5997) title and body and make sure I correctly referenced it in the above PRs body.

File: docs/admin/kubernetes-ops/tui-k8s.md
Comment (moabu) [inline docs/admin/kubernetes-ops/tui-k8s.md]: Use `vreplace-janssen-version` instead of `v1.0.17.nightly` so it gets substituted automatically

**Review by moabu — CHANGES_REQUESTED** [review]

**Comment by moabu** [comment]
i'm adding a video to this doc . I'll merge right after.

**Review by moabu — APPROVED** [review]

**Pull request merged**
